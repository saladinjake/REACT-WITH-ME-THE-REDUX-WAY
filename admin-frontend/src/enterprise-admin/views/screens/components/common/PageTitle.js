import React, { Component } from 'react';
import { Container, Row, Col } from 'react-bootstrap';
//from "./styles/pagetitle.js";

export class PageTitle extends Component {
    state = {
        backgroundImage: 'qbreadcrumb.jpg',
    }

    render() {
        return (
            <>
                <section className="breadcrumb-area" style={{ backgroundImage: `url(${process.env.PUBLIC_URL}/assets/images/${this.state.backgroundImage})` }}>
                    <Container>
                        <Row>
                            <Col md="12" className="text-left">
                                <div className="breadcrumb-box">
                                    <h2 className="breadcrumb-title">Search our courses</h2>
                                    
                                </div>
                            </Col>
                        </Row>
                    </Container>
                </section>
            </>
        )
    }
}