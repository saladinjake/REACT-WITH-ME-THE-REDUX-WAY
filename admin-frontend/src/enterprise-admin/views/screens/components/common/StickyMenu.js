import React, { useEffect } from "react";
import { Link } from "react-router-dom";
import { Container, Row, Col } from "react-bootstrap";
//from "./styles/stickyMenu.js";
import { useHistory } from "react-router-dom";

function StickyMenu({ cart, isAuthenticated, user, logOut }) {
  let history = useHistory();

  useEffect(() => {
    window.addEventListener("scroll", () => {
      const stickyMenu = document.querySelector(".sticky-menu");

      if (window.scrollY > 160) {
        stickyMenu.classList.add("sticky");
      } else {
        stickyMenu.classList.remove("sticky");
      }
    });
  });

  const handleLogout = async () => {
    await logOut();
    history.push("/login");
  };

  return (
    <>
      {/* Sticky Menu */}
      <section className="sticky-menu">
        <Container>
          <Row>
            <Col md="3">
              <div className="logo">
                <Link to={process.env.PUBLIC_URL + "/"}>
                  <img
                    src={
                      process.env.PUBLIC_URL +
                      "/assets/images/questence-logo.png"
                    }
                    alt=""
                    width="171px"
                  />
                </Link>
              </div>
            </Col>
            <Col md="9">
              <div className="menu-box d-flex justify-content-end">
                <ul className="nav menu-nav">
                  <li className="nav-item dropdown">
                    <Link
                      className="nav-link dropdown-toggle"
                      to={process.env.PUBLIC_URL + "/"}
                      data-toggle="dropdown"
                    >
                      Courses <i className="las la-angle-down"></i>
                    </Link>
                    <ul className="dropdown list-unstyled">
                      <li className="nav-item">
                        <Link
                          className="nav-link"
                          to={process.env.PUBLIC_URL + "/courses"}
                        >
                          Courses
                        </Link>
                      </li>
                      <li className="nav-item">
                        <Link
                          className="nav-link"
                          to={process.env.PUBLIC_URL + "/courses/category/1"}
                        >
                          Technology
                        </Link>
                      </li>
                      <li className="nav-item">
                        <Link
                          className="nav-link"
                          to={process.env.PUBLIC_URL + "/courses/category/4"}
                        >
                          Engineering
                        </Link>
                      </li>
                    </ul>
                  </li>
                  <li className="nav-item">
                    <Link
                      className="nav-link dropdown-toggle"
                      to={process.env.PUBLIC_URL + "/instructors"}
                    >
                      For Institutions <i className="las la-angle-down"></i>
                    </Link>
                    <ul className="dropdown list-unstyled">
                      <li className="nav-item">
                        <Link
                          className="nav-link"
                          to={process.env.PUBLIC_URL + "/"}
                        >
                          For Corporates{" "}
                        </Link>
                      </li>
                      <li className="nav-item">
                        <Link
                          className="nav-link"
                          to={process.env.PUBLIC_URL + "/institutions"}
                        >
                          For Institutions{" "}
                        </Link>
                      </li>
                      <li className="nav-item">
                        <Link
                          className="nav-link"
                          to={process.env.PUBLIC_URL + "/instructors"}
                        >
                          For Instructors{" "}
                        </Link>
                      </li>
                    </ul>
                  </li>

                  {/* <li className="nav-item">
                    <Link className="nav-link" to={"/cart"}>
                      Cart {cart !== undefined && `(${cart?.length})`}{" "}
                    </Link>
                  </li> */}
                </ul>

                {/* Idea */}
                <ul className="nav menu-nav">
                  {isAuthenticated && (
                    <li className="nav-item dropdown">
                      <Link
                        className="nav-link dropdown-toggle useraccount"
                        to={process.env.PUBLIC_URL + "/"}
                        data-toggle="dropdown"
                      >
                        <figure>
                          <img
                            src={
                              process.env.PUBLIC_URL +
                              "/assets/images/questone.jpg"
                            }
                            alt=""
                            width="20px"
                          />
                        </figure>
                        {`${user?.first_name} ${user?.last_name}`}
                      </Link>
                      <ul className="dropdown list-unstyled">
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to={process.env.PUBLIC_URL + "/dashboard"}
                          >
                            Dashboard
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to={process.env.PUBLIC_URL + "/learner/profile"}
                          >
                            Profile
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to={process.env.PUBLIC_URL + "/products"}
                          >
                            My learning
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link className="nav-link" to={"/cart"}>
                            Cart {cart !== undefined && `(${cart?.length})`}{" "}
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to={process.env.PUBLIC_URL + "/"}
                          >
                            Notifications
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to={process.env.PUBLIC_URL + "/"}
                          >
                            Messages
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to={process.env.PUBLIC_URL + "/"}
                          >
                            Billing
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to="#"
                            onClick={handleLogout}
                          >
                            Logout
                          </Link>
                        </li>
                      </ul>
                    </li>
                  )}
                </ul>
                {/* Idea */}

                <div className="apply-btn">
                  {isAuthenticated ? (
                    ""
                  ) : (
                    <Link
                      to={process.env.PUBLIC_URL + "/register"}
                      className="outline"
                    >
                      Register
                    </Link>
                  )}
                </div>
                <div className="apply-btn">
                  {isAuthenticated ? (
                    ""
                  ) : (
                    <Link to={process.env.PUBLIC_URL + "/login"}>
                      <i className="las la-user"></i>Log In
                    </Link>
                  )}
                </div>
              </div>
            </Col>
          </Row>
        </Container>
      </section>
    </>
  );
}

export default StickyMenu;
