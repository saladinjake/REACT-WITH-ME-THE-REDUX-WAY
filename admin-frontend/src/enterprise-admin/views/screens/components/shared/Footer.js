import React, { Component } from 'react';
// import { Trans } from 'react-i18next';

class Footer extends Component {
  render () {
    return (
      <footer className="footer">
        <div className="container-fluid">
          <div className="d-sm-flex justify-content-center justify-content-sm-between py-2 w-100">
            <span className="text-muted text-center text-sm-left d-block d-sm-inline-block">Copyright Â© <a href="https://www.somecompanyname.com/" target="_blank" rel="noopener noreferrer">saladin jake victor  </a>2020</span>
        
          </div>
        </div>
      </footer>
    );
  }
}

export default Footer;