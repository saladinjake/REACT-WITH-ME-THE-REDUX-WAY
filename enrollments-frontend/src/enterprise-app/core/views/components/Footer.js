import React, { Component } from 'react';
import Datas from '../data/footer/footer.json';
import { Link } from 'react-router-dom';
import { Container, Row, Col } from 'react-bootstrap';
import BackToTop from './common/BackToTop';
import { Styles } from "./styles/footerOne.js";

class Footer extends Component {
    render() {
        return (
            <Styles>
                {/* Footer Area */}
                <footer className="footer1" style={{ background: `#fff` }}>
                    <Container>
                        <Row>
                            <Col md="8">
                                <div className="footer-logo-info">
                                    <img src={process.env.PUBLIC_URL + "/assets/images/f-logo.png"} alt="" className="img-fluid" />
                                    <p>EMS ONLINE LEARNING. Grow your skills, learn and earn</p>
                                    <ul className="list-unstyled">
                                        <li><i className="las la-map-marker"></i>795 Moon Avenue, Jupiter </li>
                                        <li><i className="las la-envelope"></i>juwavictor@gmail.com</li>
                                        <li><i className="las la-phone"></i>+2348130870416</li>
                                    </ul>
                                </div>
                            </Col>
                            <Col md="4">
                                <div className="f-links">
                                    <h5>Useful Links</h5>
                                    <ul className="list-unstyled">
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Home</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Help Center</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Our Services</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Privacy Policy</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Online Support</Link></li>
                                    </ul>
                                    <ul className="list-unstyled">
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>About Us</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Courses</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Terms and conditions</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Why EMS</Link></li>
                                        <li><Link to={process.env.PUBLIC_URL + "/"}><i className="las la-angle-right"></i>Online Support</Link></li>
                                    </ul>
                                </div>
                            </Col>
                            </Row>
                    </Container>
                </footer>

                {/* Copyright Area */}
                <section className="copyright-area">
                    <Container>
                        <Row>
                            <Col md="12">
                                <div className="copy-text">
                                    <p>Copyright &copy; 2022 | Designed With <i className="las la-heart"></i> by <a href={process.env.PUBLIC_URL + "/"} target="_blank" rel="noopener noreferrer">Juwa Victor</a></p>
                                </div>
                            </Col>
                            
                        </Row>
                    </Container>

                    {/* Back To Top */}
                    <BackToTop/>
                </section>
            </Styles>
        )
    }
}

export default Footer
