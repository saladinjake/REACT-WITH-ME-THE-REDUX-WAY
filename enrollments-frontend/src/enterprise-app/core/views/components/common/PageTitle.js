import React, { useState} from 'react';
import { Container, Row, Col } from 'react-bootstrap';
import { Styles } from "./styles/pagetitle.js";
/*
*@companyName: EMS
*@Location : Lagos Nigeria
*@Author/Developer : juwa victor/saladinjake
*@AuthorsEmail : juwavictor@gmail.com
*@description: page title component

*/
const PageTitle = () =>{
    
    const [backgroundImage, setBg] = useState('qbreadcrumb.jpg')

    
        return (
            <Styles>
                <section className="breadcrumb-area" style={{ backgroundImage: `url(${process.env.PUBLIC_URL}/assets/images/${backgroundImage})` }}>
                    <Container>
                        <Row>
                            <Col md="12" className="text-left">
                                <div className="breadcrumb-box">
                                    <h2 className="breadcrumb-title">Search our courses</h2>
                                    
                                </div>
                            </Col>
                        </Row>
                    </Container>
                </section>
            </Styles>
        )
    
}

export default PageTitle;