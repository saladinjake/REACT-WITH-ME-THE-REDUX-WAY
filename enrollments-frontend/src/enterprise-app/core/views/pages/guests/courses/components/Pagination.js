import React from "react";
import { Styles } from "../styles/pagination.js";

const Pagination = ({ coursePerPage, totalCourses, paginate, currentPage }) => {
  const pageNumbers = [];

  for (let i = 1; i <= Math.ceil(totalCourses.length / coursePerPage); i++) {
    pageNumbers.push(i);
  }

  const execPaginate = (i, num) => {
    paginate(num);
  };
  return (
    <Styles>
      {/* Pagination */}
      <ul className="pagination-box list-unstyled list-inline">
        {pageNumbers.map((number, index) => {
          return (
            <li
              className={
                index + 1 === currentPage
                  ? "list-inline-item active"
                  : "list-inline-item"
              }
              key={number}
              onClick={() => execPaginate(index + 1, number)}
            >
              {number}
            </li>
          );
        })}
      </ul>
    </Styles>
  );
};

export default Pagination;
