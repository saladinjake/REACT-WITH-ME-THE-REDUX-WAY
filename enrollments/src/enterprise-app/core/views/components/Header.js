import React from "react";
import { Link } from "react-router-dom";
import { Container, Row, Col } from "react-bootstrap";
import Search from "./common/Search";
import StickyMenu from "./common/StickyMenu";
import MobileMenu from "./common/MobileMenu";
import { Styles } from "./styles/header.js";

import PropTypes from "prop-types";
import { connect } from "react-redux";
import { logOut } from "../../redux/actions/authActions";

const HeaderTwo = ({ cart: { cart }, auth: { isAuthenticated }, logOut }) => {
  return (
    <Styles>
      {/* Topbar 2 */}

      {/* Logo Area 2 */}
      <section className="logo-area2">
        <Container>
          <Row>
            <Col md="3">
              <div className="logo">
                <Link to={process.env.PUBLIC_URL + "/"}>
                  <img
                    src={process.env.PUBLIC_URL + "/assets/images/questence-logo.png"} alt="" width="171px"/></Link>
              </div>
            </Col>
            <Col md="9">
              <div className="menu-box d-flex justify-content-end">
                <ul className="nav menu-nav">
                  {isAuthenticated && (
                    <li className="nav-item dropdown">
                      <Link
                        className="nav-link dropdown-toggle"
                        to={process.env.PUBLIC_URL + "/"}
                        data-toggle="dropdown"
                      >
                        Account <i className="las la-angle-down"></i>
                      </Link>
                      <ul className="dropdown list-unstyled">
                        <li className="nav-item">
                          <Link
                            className="nav-link"
                            to={process.env.PUBLIC_URL + "/products"}
                          >
                            Dashboard
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link className="nav-link" to={"/cart"}>
                            Cart {cart !== undefined && `(${cart?.length})`}{" "}
                          </Link>
                        </li>
                        <li className="nav-item">
                          <Link className="nav-link" to="#" onClick={logOut}>
                            Logout
                          </Link>
                        </li>
                      </ul>
                    </li>
                  )}

                  <li className="nav-item dropdown">
                    <Link
                      className="nav-link "
                      to={process.env.PUBLIC_URL + "/"}
                     
                    >
                      Home <i className="las la-angle-down"></i>
                    </Link>
                    
                  </li>
                  <li className="nav-item dropdown">
                    <Link
                      className="nav-link dropdown-toggle"
                      to={process.env.PUBLIC_URL + "/"}
                      data-toggle="dropdown"
                    >
                      Categories <i className="las la-angle-down"></i>
                    </Link>
                    <ul className="dropdown list-unstyled">
                      <li className="nav-item">
                        <Link
                          className="nav-link"
                          to={process.env.PUBLIC_URL + "/courses"}
                        >
                          All Courses
                        </Link>
                      </li>
                      
                    </ul>
                  </li>
                  
                  <li className="nav-item ">
                    <Link
                      className="nav-link"
                      to={process.env.PUBLIC_URL + "/cart"}
                    >
                      Cart {` (${cart.length})`}
                    </Link>
                  </li>
                </ul>
                <div className="search-box">
                  <Search />
                </div>
                <div className="apply-btn">
                  <Link to={process.env.PUBLIC_URL + "/login"}> <i className="las la-user"></i>Log In</Link>
                </div>
              </div>
            </Col>
          </Row>
        </Container>
      </section>

      {/* Sticky Menu */}
      <StickyMenu
        isAuthenticated={isAuthenticated}
        logOut={logOut}
        cart={cart}
      />

      {/* Mobile Menu */}
      <MobileMenu />
    </Styles>
  );
};

// export default HeaderTwo;
HeaderTwo.propTypes = {
  auth: PropTypes.object.isRequired,
  logOut: PropTypes.func.isRequired,
  cart: PropTypes.object.isRequired,
};

const mapStateToProps = (state) => ({
  cart: state.cart,
  auth: state.auth,
});

export default connect(mapStateToProps, { logOut })(HeaderTwo);
