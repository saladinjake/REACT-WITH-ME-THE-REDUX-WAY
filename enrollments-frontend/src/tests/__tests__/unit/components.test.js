/*@testCases: test individual function
 *@testType: unit test
 *@testCategory: simple
 *@author: saladinjake victor
 *@company: Ficticious EMS
 *@date" 5/20/2022
*/
import React from "react"
import ReactDOM from 'react-dom';
import { 
  BrowserRouter , 
  Switch, 
  Route 
} from "react-router-dom";
import { shallow , mount } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
configure({ adapter: new Adapter() });
window.matchMedia = window.matchMedia || function() {
  return {
    matches : false,
    addListener : function() {},
    removeListener: function() {}
  };
};


import Footer from "../../../enterprise-app/core/views/components/Footer"
import CourseSlider from "../../../enterprise-app/core/views/components/CourseSlider"
import FreeCourse from "../../../enterprise-app/core/views/components/FreeCourse"
import Header from "../../../enterprise-app/core/views/components/Header"
import HeroImage from "../../../enterprise-app/core/views/components/HeroImage"
import HeroSlider from "../../../enterprise-app/core/views/components/HeroSlider"
import Pagination from "../../../enterprise-app/core/views/components/Pagination"
import QuestOne from "../../../enterprise-app/core/views/components/QuestOne"
import ServiceBox from "../../../enterprise-app/core/views/components/ServiceBox"



import BackToTop from "../../../../src/enterprise-app/core/views/components/common/BackToTop"
import BreadcrumbBox from "../../../../src/enterprise-app/core/views/components/common/Breadcrumb"
import MobileMenu from "../../../../src/enterprise-app/core/views/components/common/MobileMenu"
import PageTitle from "../../../../src/enterprise-app/core/views/components/common/PageTitle"
import Search from "../../../../src/enterprise-app/core/views/components/common/Search"
import Sidebar from "../../../../src/enterprise-app/core/views/components/common/Sidebar"
import StickyMenu from "../../../../src/enterprise-app/core/views/components/common/StickyMenu"


// import App from '../../../../src/App';


// describe('App Component', () => {
//   let wrapper;
//   beforeEach(() => {
//     wrapper = shallow(<App />);
//   });

//   it('should match snapshot', () => {
//     expect(wrapper).toMatchSnapshot();
//   });
//   // it('renders without crashing', () => {
//   //   const div = document.createElement('div');
//   //   ReactDOM.render(<App />, div);
//   //   ReactDOM.unmountComponentAtNode(div);
//   // });
// });

//snake in the monkey shadow
describe("Footer  in the dom shadow", () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find('.footer1').exists()).toBe(true);


  });

  
});



describe("CourseSlider  in the dom shadow", () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<CourseSlider />);
     expect(wrapper).toMatchSnapshot();
  });

  
});



describe("FreeCourse  in the dom shadow", () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<FreeCourse />);
     expect(wrapper).toMatchSnapshot();
  });

 });


describe("Header  in the dom shadow", () => {
  it('should render without crashing', () => {
    //for nested components just use the naked butt
    const wrapper = <Header />;
     expect(wrapper).toMatchSnapshot();
  });

});

describe("HeroImage  in the dom shadow", () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<HeroImage />);
    expect(wrapper.find('.hero-image-area').exists()).toBe(true);
  });
});

describe("HeroSlider  in the dom shadow", () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<HeroSlider />);
    expect(wrapper.find('.hero-slider-area').exists()).toBe(true);
  });

});

describe("Pagination  in the dom shadow", () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Pagination />);
    expect(wrapper.find('.pagination-box').exists()).toBe(true);
  });
});

describe("ServiceBox  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    const wrapper = shallow(<ServiceBox />);
     expect(wrapper).toMatchSnapshot();
  });
});





describe("BackToTop  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    const wrapper = shallow(<BackToTop />);
     expect(wrapper).toMatchSnapshot();
  });
});


describe("Breadcrumb  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    //for nested components just use the naked butt
    const wrapper = <BreadcrumbBox />;
    expect(wrapper).toMatchSnapshot();
  });
  
});


describe("MobileMenu  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    const wrapper = shallow(<MobileMenu />);
    expect(wrapper.find('.mobile-menu-area').exists()).toBe(true);
  });
});


describe("PageTitle  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    const wrapper = shallow(<PageTitle />);
    expect(wrapper).toMatchSnapshot();
  });
});


describe("Search  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    const wrapper = shallow(<Search />);
    expect(wrapper.find('.nav-search').exists()).toBe(true);
  });
});


describe("StickyMenu  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    const wrapper = shallow(<Sidebar />);
    expect(wrapper.find('.sidebar').exists()).toBe(true);
  });
});
describe("StickyMenu  in the dom shadow", () => {
  it('should render ServiceBox without crashing', () => {
    const wrapper = shallow(<StickyMenu />);
    expect(wrapper.find('.sticky-menu').exists()).toBe(true);
  });
});