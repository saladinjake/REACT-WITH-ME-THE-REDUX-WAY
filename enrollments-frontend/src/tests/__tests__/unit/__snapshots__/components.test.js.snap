// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackToTop  in the dom shadow should render ServiceBox without crashing 1`] = `ShallowWrapper {}`;

exports[`Breadcrumb  in the dom shadow should render ServiceBox without crashing 1`] = `<BreadcrumbBox />`;

exports[`CourseSlider  in the dom shadow should render without crashing 1`] = `ShallowWrapper {}`;

exports[`FreeCourse  in the dom shadow should render without crashing 1`] = `ShallowWrapper {}`;

exports[`Header  in the dom shadow should render without crashing 1`] = `<UNDEFINED />`;

exports[`PageTitle  in the dom shadow should render ServiceBox without crashing 1`] = `ShallowWrapper {}`;

exports[`ServiceBox  in the dom shadow should render ServiceBox without crashing 1`] = `ShallowWrapper {}`;
